modules/efk-logging-fluentd-json.adoc

```diff
@@ -40,29 +42,31 @@ You can use link:https://github.com/openshift/origin-aggregated-logging/blob/mas
 
 However, if you set the `CDM_UNDEFINED_TO_STRING` enviroment variable to `true` when `MERGE_JSON_LOG` is also set to `true`, you will receive an Elasticsearch *400* error if you have already generated conflicting fields, until the indices roll over for the next day. The error occurs because when`MERGE_JSON_LOG=true`, Fluentd adds fields with data types other than *string*. When you set `CDM_UNDEFINED_TO_STRING=true`, Fluentd attempts to add those fields with a *string* value resulting in the Elasticsearch *400* error. 
 
-When Fluentd rolls over the indices for the next day's logs, it will create a brand new index. The field definitions are updated and you will not get the *400* error. The only thing that can be done is to use something like https://github.com/richm/docs/releases/tag/20180510180102 - the section about @ERROR Error record handling:
+When Fluentd rolls over the indices for the next day's logs, it will create a brand new index. The field definitions are updated and you will not get the *400* error. 
 
 Records that have "hard" errors (schema violations, corrupted data, etc.) that
-cannot be retried will be sent to the @ERROR handler. If you add a
-<label @ERROR> section to your fluentd config, you can handle these records
+cannot be retried will be sent to the @ERROR handler. If you link:https://docs.fluentd.org/v1.0/articles/config-file#@error-label[add a
+`<label @ERROR>` section] to your fluentd config, you can handle these records
 as needed. 
 
 For example:
 
 ----
-<label @ERROR>
-  <match **>
-    @type file
-    path /var/log/fluent/dlq
-    time_slice_format %Y%m%d
-    time_slice_wait 10m
-    time_format %Y%m%dT%H%M%S%z
-    compress gzip
-  </match>
-</label>
+data:
+  fluent.conf:
+    <label @ERROR>
```

Comment:
we should state to add this to the existing data fluent.conf after the last line (after the last `</label>`)